integration/firestore/firebase_export.ts

```diff
@@ -33,7 +33,10 @@ export function newTestFirestore(
   }
   const app =
     typeof nameOrApp === 'string'
-      ? firebase.initializeApp({ apiKey: 'fake-api-key', projectId }, nameOrApp)
+      ? firebase.initializeApp(
+          { apiKey: 'A-test-api-key-jklmnopqrstuvwxyz1234567', projectId },
```

Comment:
Does the test fail without the change? I think you only need the correct format in installation tests, specifically the tests that involves `extractAppConfig()`. We can leave other places unchanged, right?

Reply:
Yeah, I saw many tests were failing so I added these changes.

Reply:
That is weird. I don't think your change affects firestore. Are you sure the failure comes from firestore? It will be really inconvenient if we are forced use exact 38 characters apiKey in tests.